vmware_dvs/utils/dvs_util.py

```diff
@@ -285,8 +286,23 @@ def _build_pg_update_spec(self, config_version,
         pg.configVersion = config_version
         return pg
 
-    def _get_dvs(self, dvs_name, connection):
+    def _get_dvs(self, dvs_name, cluster_name, connection):
         """Get the dvs by name"""
+        cluster = None
+        clusters = self.connection.invoke_api(
+            vim_util, 'get_objects', self.connection.vim,
+            'ClusterComputeResource', 100, ['name']).objects
```

Comment:
100 - is it a parameter for limitation of objects count? could we use named constant instead of magic number?